ray-operator/config/samples/ray_v1alpha1_rayservice.yaml

```diff
@@ -3,6 +3,8 @@ kind: RayService
 metadata:
   name: rayservice-sample
 spec:
+  serviceUnhealthySecondThreshold: 300
+  deploymentUnhealthySecondThreshold: 300
```

Comment:
Are these required fields?
If they're not, are these the defaults?

It would be good to add some comments to the example config describing what each field means.

Reply:
(Even if the fields are already documented elsewhere)